testing/conftest.py

```diff
@@ -226,24 +225,8 @@ def mock_timing(monkeypatch: MonkeyPatch):
     Time is static, and only advances through `sleep` calls, thus tests might sleep over large
     numbers and obtain accurate time() calls at the end, making tests reliable and instant.
     """
-
-    @dataclasses.dataclass
-    class MockTiming:
-        _current_time: float = 1590150050.0
-
-        def sleep(self, seconds: float) -> None:
-            self._current_time += seconds
-
-        def time(self) -> float:
-            return self._current_time
-
-        def patch(self) -> None:
-            from _pytest import timing
-
-            monkeypatch.setattr(timing, "sleep", self.sleep)
-            monkeypatch.setattr(timing, "time", self.time)
-            monkeypatch.setattr(timing, "perf_counter", self.time)
+    from _pytest.timing import MockTiming
```

Comment:
Does it really make sense to have this moved into `_pytest.timing` rather than having it in the testsuite? I suppose it's a bit of a special case because the whole point of `timing.py` is testability I suppose?

Reply:
as a followup it could be exposed as a opt-in in pytester so external plugins could work with the time pytest sees